src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -0,0 +1,103 @@
+  --pf-c-app-launcher__toggle--Color: var(--pf-global--Color--200);
+  --pf-c-app-launcher__toggle--hover--Color: var(--pf-global--Color--100);
+  --pf-c-app-launcher__toggle--active--Color: var(--pf-global--Color--100);
+
+  // Menu
+  --pf-c-app-launcher__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-app-launcher__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-app-launcher__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-app-launcher__menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu--Top: calc(100% + var(--pf-global--spacer--xs));
+  --pf-c-app-launcher__menu--ZIndex: var(--pf-global--ZIndex--xs);
+
+  // Menu Item
+  --pf-c-app-launcher__menu-item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu-item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu-item--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-item--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-app-launcher__menu-item--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-app-launcher__menu-item--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-app-launcher__menu-item--Color: var(--pf-global--Color--dark-100);
+  --pf-c-app-launcher__menu-item--disabled--Color: var(--pf-global--Color--dark-200);
+  --pf-c-app-launcher__menu-item--hover--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+
+  position: relative;
+  display: inline-block;
+  max-width: 100%;
+}
+
+.pf-c-app-launcher__toggle {
+  display: flex;
+  justify-content: center;
+  min-width: var(--pf-c-app-launcher__toggle--MinWidth);
+  padding: var(--pf-c-app-launcher__toggle--PaddingTop) var(--pf-c-app-launcher__toggle--PaddingRight) var(--pf-c-app-launcher__toggle--PaddingBottom) var(--pf-c-app-launcher__toggle--PaddingLeft);
+  background-color: var(--pf-c-app-launcher__toggle--BackgroundColor);
+  border: none;
+
+  .pf-c-app-launcher__toggle-icon {
+    color: var(--pf-c-app-launcher__toggle--Color);
+
+    &:hover,
+    &.pf-m-hover {
+      --pf-c-app-launcher__toggle--Color: var(--pf-c-app-launcher__toggle--hover--Color);
+    }
+
+    &:active,
+    &.pf-m-active,
+    .pf-m-expanded > & {
+      --pf-c-app-launcher__toggle--Color: var(--pf-c-app-launcher__toggle--active--Color);
+    }
+  }
+}
+
+.pf-c-app-launcher__menu {
+  position: absolute;
+  top: var(--pf-c-app-launcher__menu--Top);
+  z-index: var(--pf-c-app-launcher__menu--ZIndex);
+  min-width: 100%;
+  padding-top: var(--pf-c-app-launcher__menu--PaddingTop);
+  padding-bottom: var(--pf-c-app-launcher__menu--PaddingBottom);
+  background: var(--pf-c-app-launcher__menu--BackgroundColor);
+  background-clip: padding-box;
+  border: var(--pf-c-app-launcher__menu--BorderWidth) solid transparent;
```

Comment:
do you know why we set this?

Reply:
this is another thing I want to ask her. I've noticed with the dropdown that it isn't the right width and looks like it's caused by this. I want to ask her about it b/c this seems really intentional.

Reply:
possibly due to high contrast mode. I will investigate further.